### DISCUSSION TITLE
代码生成器，生成的迁移文件路径问题

### DISCUSSION THREAD

**Discussion by YouLiLiao** [slowlyo/owl-admin#111] [FAQ]
迁移文件的路径不以Laravel的路径为主吗？像这样生成之后，运行php artisan migrate:status之后找不到迁移文件
![89d222f008525b3e039f93182b00936](https://github.com/slowlyo/owl-admin/assets/11286354/fda35ea6-242b-4149-b3ec-0a1fe41c6d37)
![1a12adec33dc1bac59d723bdfda52bb](https://github.com/slowlyo/owl-admin/assets/11286354/09bdfa56-435c-4033-ad3d-ae2b3624d691)
![c2e96d675a9ce20ced70234b388a309](https://github.com/slowlyo/owl-admin/assets/11286354/a23be300-a973-4481-b442-38cbd0c02a6a)

**Comment by slowlyo** [comment]
点生成了么..

**Reply by YouLiLiao** [reply]
点了，因为我改过应用安装目录，所以生成的路径不对
![image](https://github.com/slowlyo/owl-admin/assets/11286354/600a4169-de33-4b84-92a3-74c9cb276cea)

**Reply by YouLiLiao** [reply]
看下是否使用绝对路径来解决这个问题
![image](https://github.com/slowlyo/owl-admin/assets/11286354/b644a9a3-85bf-4117-a4b1-d33f487334ab)

**Reply by slowlyo** [reply]
这里确实不太严谨, 后续我优化下

**Reply by YouLiLiao** [reply]
应为改过应用安装目录，所以代码生成器的默认路径都跟着变了，这里再用相对路径就会导致这个问题。：）
![image](https://github.com/slowlyo/owl-admin/assets/11286354/3023f5c3-f7bc-47e3-8b5f-6c016ed369a4)

**Reply by YouLiLiao** [reply]
一开始没注意看这个路径，现在知道是什么问题了

**Reply by YouLiLiao** [reply]
![image](https://github.com/slowlyo/owl-admin/assets/11286354/6f815e31-4b37-420c-b924-00ecf66e579c)
![image](https://github.com/slowlyo/owl-admin/assets/11286354/ff2ee5df-474b-4d69-ab82-58e4d0fa13f4)
每次改完，再点编辑的时候，回显的时候又变回原来的了。T^T

**Comment by slowlyo** [comment]
`v3.6.10` 修复了这两个问题~

**Reply by YouLiLiao** [reply]
我刚才测试了一下
![image](https://github.com/slowlyo/owl-admin/assets/11286354/c1db7f26-ab6d-4893-be69-0c7173fac3e8)

**Reply by slowlyo** [reply]
删了重发了

**Reply by YouLiLiao** [reply]
![image](https://github.com/slowlyo/owl-admin/assets/11286354/1f10b94e-a757-4b40-aec1-6b88b6e2e3f3)
哈哈哈，哥哥，别搞事啊

**Reply by slowlyo** [reply]
卧槽

**Reply by slowlyo** [reply]
删了重发了 :joy:
